### PR TITLE
translate api work resource

### PR DESCRIPTION
**Pull request by XiShanYongYe-Chang**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

translate API work resource

**Which issue(s) this PR fixes**:
Part of #483 

**Special notes for your reviewer**:

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md [hunk 1]
```diff
@@ -30,559 +30,559 @@ ResourceBinding represents a binding of a kubernetes resource with a propagation
...
-- **spec** ([ResourceBindingSpec](../work-resources/resource-binding-v1alpha2#resourcebindingspec)), required
+- **spec** ([ResourceBindingSpec](../work-resources/resource-binding-v1alpha2#resourcebindingspec))，必选
 
-  Spec represents the desired behavior.
+  Spec表示规范。
 
 - **status** ([ResourceBindingStatus](../work-resources/resource-binding-v1alpha2#resourcebindingstatus))
 
-  Status represents the most recently observed status of the ResourceBinding.
+  Status表示ResourceBinding的最新状态。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md:39]:
```suggestion
  status表示ResourceBinding的最新状态。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md [hunk 2]
```diff
@@ -30,559 +30,559 @@ ResourceBinding represents a binding of a kubernetes resource with a propagation
...
-ResourceBindingSpec represents the expectation of ResourceBinding.
+ResourceBindingSpec表示预期的ResourceBinding。
 
 <hr/>
 
-- **resource** (ObjectReference), required
+- **resource** (ObjectReference)，必选
 
-  Resource represents the Kubernetes resource to be propagated.
+  Resource表示要传播的Kubernetes资源。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md:49]:
```suggestion
  resource表示要传播的Kubernetes资源。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md [hunk 3]
```diff
@@ -30,559 +30,559 @@ ResourceBinding represents a binding of a kubernetes resource with a propagation
...
+  - **resource.apiVersion**（string），必选
 
-    APIVersion represents the API version of the referent.
+    **apiVersion**表示所引用资源的API版本。
 
-  - **resource.kind** (string), required
+  - **resource.kind**（string），必选
 
-    Kind represents the Kind of the referent.
+    Kind表示所引用资源的类别。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md:61]:
```suggestion
    kind表示所引用资源的类别。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md [hunk 4]
```diff
@@ -30,559 +30,559 @@ ResourceBinding represents a binding of a kubernetes resource with a propagation
...
 
-  - **resource.name** (string), required
+  - **resource.name**（string），必选
 
-    Name represents the name of the referent.
+    Name表示所引用资源的名称。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md:65]:
```suggestion
    name表示所引用资源的名称。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md [hunk 5]
```diff
@@ -30,559 +30,559 @@ ResourceBinding represents a binding of a kubernetes resource with a propagation
...
 
-  - **resource.namespace** (string)
+  - **resource.namespace**（string）
 
-    Namespace represents the namespace for the referent. For non-namespace scoped resources(e.g. 'ClusterRole')，do not need specify Namespace, and for namespace scoped resources, Namespace is required. If Namespace is not specified, means the resource is non-namespace scoped.
+    Namespace表示所引用资源所在的命名空间。对于非命名空间范围的资源（例如ClusterRole），不需要指定命名空间。对于命名空间范围的资源，需要指定命名空间。如果未指定命名空间，则资源不在命名空间范围内。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md:69]:
```suggestion
    namespace表示所引用资源所在的命名空间。对于非命名空间范围的资源（例如ClusterRole），不需要指定命名空间。对于命名空间范围的资源，需要指定命名空间。如果未指定命名空间，则资源不在命名空间范围内。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md [hunk 6]
```diff
@@ -30,559 +30,559 @@ ResourceBinding represents a binding of a kubernetes resource with a propagation
...
 
-  - **resource.resourceVersion** (string)
+  - **resource.resourceVersion**（string）
 
-    ResourceVersion represents the internal version of the referenced object, that can be used by clients to determine when object has changed.
+    ResourceVersion表示所引用资源的内部版本，客户端可用其确定资源的变更时间。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md:73]:
```suggestion
    resourceVersion表示所引用资源的内部版本，客户端可用其确定资源的变更时间。
```

**Review by windsonsea — COMMENTED** [review]
The translation is good enough. Pick some issues about style:

- Space issue. You can try to use [prettier](https://prettier.io/docs/en/install) to change it with one command like `npx prettier --w i18n/zh/docusa....`
- Capitalization concern. Some commands are capitalized by auto-generation in en version.

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md [hunk 7]
```diff
@@ -30,559 +30,559 @@ ResourceBinding represents a binding of a kubernetes resource with a propagation
 
 - **metadata** ([ObjectMeta](../common-definitions/object-meta#objectmeta))
 
-- **spec** ([ResourceBindingSpec](../work-resources/resource-binding-v1alpha2#resourcebindingspec)), required
+- **spec** ([ResourceBindingSpec](../work-resources/resource-binding-v1alpha2#resourcebindingspec))，必选
 
-  Spec represents the desired behavior.
+  Spec表示规范。
```
[the remark this answers is not in the document]
Comment (XiShanYongYe-Chang) [reply i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md:35]: 我同意你的观点，需要与代码保持一致，多谢指出。

**Comment by windsonsea** [comment]
Thanks.
/lgtm

**Comment by XiShanYongYe-Chang** [comment]
Thanks @windsonsea 
/cc @RainbowMango

**Comment by XiShanYongYe-Chang** [comment]
Hi @windsonsea, I have updated all three files, can you help retake a review again?

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/cluster-resource-binding-v1alpha2.md [hunk 8]
```diff
@@ -18,9 +18,9 @@ auto_generated: true
 
 `import "github.com/karmada-io/karmada/pkg/apis/work/v1alpha2"`
 
-## ClusterResourceBinding 
+## ClusterResourceBinding
 
-ClusterResourceBinding represents a binding of a kubernetes resource with a ClusterPropagationPolicy.
+ClusterResourceBinding 表示某种 Kubernetes 资源与集群传播策略（ClusterPropagationPolicy）之间的绑定关系。
```
[the remark this answers is not in the document]
Comment (XiShanYongYe-Chang) [reply i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/cluster-resource-binding-v1alpha2.md]: Updated.

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md [hunk 9]
```diff
@@ -30,582 +30,577 @@ ResourceBinding represents a binding of a kubernetes resource with a propagation
...
-ResourceBindingSpec represents the expectation of ResourceBinding.
+ResourceBindingSpec 表示预期的 ResourceBinding。
 
 <hr/>
 
-- **resource** (ObjectReference), required
+- **resource** (ObjectReference)，必选
 
-  Resource represents the Kubernetes resource to be propagated.
+  Resource 表示要传播的 Kubernetes 资源。
```
Comment (RainbowMango) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md:49]:
```suggestion
  Resource 表示要分发的 Kubernetes 资源。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md [hunk 10]
```diff
@@ -30,582 +30,577 @@ ResourceBinding represents a binding of a kubernetes resource with a propagation
...
-    Replicas in target cluster
+    Replicas 表示目标集群中的副本。
 
-- **conflictResolution** (string)
+- **conflictResolution**（string）
 
-  ConflictResolution declares how potential conflict should be handled when a resource that is being propagated already exists in the target cluster.
-  
-  It defaults to "Abort" which means stop propagating to avoid unexpected overwrites. The "Overwrite" might be useful when migrating legacy cluster resources to Karmada, in which case conflict is predictable and can be instructed to Karmada take over the resource by overwriting.
+  ConflictResolution 表示当目标集群中已存在正在传播的资源时，处理潜在冲突的方式。
```
Comment (RainbowMango) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md:97]:
```suggestion
  ConflictResolution 表示当目标集群中已存在正在分发的资源时，处理潜在冲突的方式。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md [hunk 11]
```diff
@@ -30,582 +30,577 @@ ResourceBinding represents a binding of a kubernetes resource with a propagation
...
+
+  默认为 Abort，表示停止传播资源以避免意外覆盖。将原集群资源迁移到 Karmada 时，可设置为 Overwrite。此时，冲突是可预测的，且 Karmada 可通过覆盖来接管资源。
```
Comment (RainbowMango) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md:99]:
```suggestion
  默认为 Abort，表示停止分发资源以避免意外覆盖。将原集群资源迁移到 Karmada 时，可设置为 Overwrite。此时，冲突是可预测的，且 Karmada 可通过覆盖来接管资源。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md [hunk 12]
```diff
@@ -30,582 +30,577 @@ ResourceBinding represents a binding of a kubernetes resource with a propagation
...
+  Replicas 表示应驱逐的副本数量。对于没有副本的资源类型，忽略该字段。
+
+- **gracefulEvictionTasks.suppressDeletion**（boolean）
+
+  SuppressDeletion 表示宽限期将持续存在，直至工具或人工干预为止。它不能与 GracePeriodSeconds 共存。
 
 - **placement** (Placement)
 
-  Placement represents the rule for select clusters to propagate resources.
+  Placement 表示选定群集以传播资源的规则。
```
Comment (RainbowMango) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md:190]:
```suggestion
  Placement 表示选定群集以及分发资源的规则。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md [hunk 13]
```diff
@@ -30,582 +30,577 @@ ResourceBinding represents a binding of a kubernetes resource with a propagation
...
-    - **placement.clusterTolerations.value** (string)
+    - **placement.clusterTolerations.value**（string）
 
-      Value is the taint value the toleration matches to. If the operator is Exists, the value should be empty, otherwise just a regular string.
+      Value 是容忍度匹配到的污点值。如果运算符为 Exists，则值应为空，否则就是一个普通字符串。
 
   - **placement.replicaScheduling** (ReplicaSchedulingStrategy)
 
-    ReplicaScheduling represents the scheduling policy on dealing with the number of replicas when propagating resources that have replicas in spec (e.g. deployments, statefulsets) to member clusters.
+    ReplicaScheduling 表示将 spec 中规约的副本资源（例如 Deployments、Statefulsets）传播到成员集群时处理副本数量的调度策略。
```
Comment (RainbowMango) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md:317]:
```suggestion
    ReplicaScheduling 表示将 spec 中规约的副本资源（例如 Deployments、Statefulsets）分发到成员集群时处理副本数量的调度策略。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md [hunk 14]
```diff
@@ -30,582 +30,577 @@ ResourceBinding represents a binding of a kubernetes resource with a propagation
...
+      当 ReplicaSchedulingType 设置为 Divided 时，由 ReplicaDivisionPreference 确定副本的分配策略。取值包括 Aggregated 和 Weighted。Aggregated：将副本分配给尽可能少的集群，同时考虑集群的资源可用性。Weighted：根据 WeightPreference 按权重分配副本。
 
-    - **placement.replicaScheduling.replicaSchedulingType** (string)
+    - **placement.replicaScheduling.replicaSchedulingType**（string）
 
-      ReplicaSchedulingType determines how the replicas is scheduled when karmada propagating a resource. Valid options are Duplicated and Divided. "Duplicated" duplicates the same replicas to each candidate member cluster from resource. "Divided" divides replicas into parts according to number of valid candidate member clusters, and exact replicas for each cluster are determined by ReplicaDivisionPreference.
+      ReplicaSchedulingType 确定 karmada 传播资源时副本的调度方式。取值包括 Duplicated 和 Divided。Duplicated：将相同的副本从资源复制到每个候选成员群集。Divided：根据有效候选成员集群的数量分配副本，每个集群的副本由 ReplicaDivisionPreference 确定。
```
Comment (RainbowMango) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md:329]:
```suggestion
      ReplicaSchedulingType 确定 karmada 分发资源副本的调度方式。取值包括 Duplicated 和 Divided。Duplicated：将相同的副本从资源复制到每个候选成员群集。Divided：根据有效候选成员集群的数量分配副本，每个集群的副本由 ReplicaDivisionPreference 确定。
```

**Review by RainbowMango — COMMENTED** [review]
请全量替换一下 `传播` -- > `分发`

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md (hunk 14 above)
Comment (XiShanYongYe-Chang) [reply i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md]: 已更新，对 传播 一词在当前文件夹下进行了替换，是否需要在整个仓库进行全量替换？

Comment (RainbowMango) [reply i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/work-resources/resource-binding-v1alpha2.md]: Yes, but please do that by another PR.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**